pkg/transformer/openshift/openshift_test.go

```diff
@@ -17,19 +17,20 @@ limitations under the License.
 package openshift
 
 import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
 	"os"
 	"path/filepath"
+	"reflect"
 	"testing"
 
-	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/runtime"
-
 	deployapi "github.com/openshift/origin/pkg/deploy/api"
 
 	"github.com/kubernetes-incubator/kompose/pkg/kobject"
 	"github.com/kubernetes-incubator/kompose/pkg/testutils"
 	"github.com/kubernetes-incubator/kompose/pkg/transformer/kubernetes"
 	"github.com/pkg/errors"
+	kapi "k8s.io/kubernetes/pkg/api"
```

Comment:
errr.. why are we importing this twice? "k8s.io/kubernetes/pkg/api" is already imported